nuxt.schema.ts

```diff
@@ -90,7 +90,8 @@ export default defineNuxtSchema({
            * @example 'https://exactpro.github.io/docs-toolkit'
            * @type {?string}
            */
-          baseUrl: undefined
+          baseUrl: undefined,
```

Comment:
Now this option is used just as hostname, so it is better to rename it `hostname`. Don't forget to update JSDoc comment.

Reply:
updated the schema (ran generate) and the `app.config` by replacing baseUrl with hostname but continues to say the hostname does not existing when I try to assign it to hostname in sitemap.xm.ts . So for now I kept the name as baseUrl .

Reply:
You also should change `baseUrl` to `hostname` in places where it is used.